crates/stages/stages/src/stages/execution.rs

```diff
@@ -270,6 +276,21 @@ where
             })?;
             execution_duration += execute_start.elapsed();
 
+            // Log execution throughput
+            if execution_duration - last_duration >= log_speed_duration {
```

Comment:
please use `Instant`, durations are not meant to capture moments in time. see e.g. https://doc.rust-lang.org/std/time/struct.Instant.html#method.elapsed

Reply:
@onbjerg I am using `execution_duration - last_duration` here because we should only count the elapsed time in executing. This excludes the elapsed time in fetching blocks.

Reply:
The point here is that this does not actually log every 10 seconds, instead it logs every 10 seconds of execution time. It would probably be easier to just keep a mutable  `Instant`, check `elapsed()`, and log if elapsed is greater than 10s

Reply:
Got it! Now updated.